src/shared/server-config.toit

```diff
@@ -54,6 +59,15 @@ abstract class ServerConfig:
   */
   abstract to-service-json [--der-serializer] -> Map
 
+  /**
+  A unique key that can be used for caching.
+  */
+  cache-key -> string:
+    if not cache-key_:
+      hash := md5.md5 (ubjson.encode (to-json --der-serializer=: it))
+      cache-key_ = "$name-$(base64.encode hash)"
```

Comment:
Maybe consider leading with the hash (see previous comment)? I'm not entirely sure it's better because it is also pretty common to have the hash at the end. So if we rarely need to look at these in lists, it's probably better to keep them at the end.

Reply:
done.